include/xplot/xmarks.hpp

```diff
@@ -593,6 +630,64 @@ namespace xpl
         };        
     }    
 
+    /****************************
+    * xheat_map implementation *
+    ****************************/
+
+    template <class D>
+    template <class SX, class SY, class SC>
+    inline xheat_map<D>::xheat_map(std::vector<double>& x_,
+                                   std::vector<double>& y_,
+                                   std::vector<std::vector<double>>& color_,
```

Comment:
Note: In other marks, we did not pass data attributes to the constructor (for now), we might want to do so though eventually. Is it necessary in the case of the heatmap?

Reply:
I know but without that, it doesn't work.

It seems that the implementation of heatmap in bqplot need to know what x, y and color are to be able to construct it.

Reply:
+1